web/src/beta/components/Resizable/index.tsx

```diff
@@ -33,24 +41,50 @@ const Resizable: React.FC<Props> = ({
   const BottomGutter = showBottomGutter ? <HorizontalGutter {...gutterProps} /> : null;
   const LeftGutter = showLeftGutter ? <VerticalGutter {...gutterProps} /> : null;
 
+  useEffect(() => {
+    if (size <= initialSize / 2) {
```

Comment:
`minWidth / 2` instead of `initialSize / 2` might be better.
It does not work with `size: 400, minWidth: 300` props.

or please update [storybook](https://deploy-preview-572--reearth-web.netlify.app/storybook/?path=/story/beta-components-resizable--vertical) as well

Reply:
We removed the minSize/maxSize from editor  based on previous discussion with kyle and based on requirement we need to compare with initialSize not minSize , For storybook  I modified it match this case.

Reply:
We got minWidth again in the review process, so using minWidth / 2 is better now.